src/set_theory/cardinal/finite.lean

```diff
@@ -41,6 +41,23 @@ cardinal.to_nat_congr f
 lemma card_eq_of_bijective (f : α → β) (hf : function.bijective f) : nat.card α = nat.card β :=
 card_congr (equiv.of_bijective f hf)
 
+lemma card_le_of_injective [finite β] (f : α → β) (h : function.injective f) :
+  nat.card α ≤ nat.card β :=
```

Comment:
This is https://leanprover-community.github.io/mathlib_docs/data/finite/card.html#finite.card_le_of_injective